add-steps/k8s/add-steps.yml

```diff
@@ -788,7 +852,7 @@ spec:
             -concurrency 50 \
             -interval 5s \
             -totalRequests 10000000 \
-            http://baseline-app.add-steps.svc.cluster.local:8501
+            http://baseline-app:8501
```

Comment:
@briansmith not urgent, but i would *love* to understand what's going on here...

## Background

I made this code change to ensure round robin DNS behavior on GKE. Prior to this change, `slow-cooker` would always connect to the same IP.

## Issue summary

In Kubernetes, when looking up a host like `baseline-app.add-steps.svc.cluster.local`, it returns IP addresses sorted by some RFC spec (not deterministic, but the first IP is always the same), causing slow-cooker to pin to the same IP address for all requests.

If we lookup `baseline-app` instead, it (more correctly?) returns the list of IPs in random order, allowing expected round robin behavior.

## Steps to reproduce

Run this inside a container on GKE:

```go
package main

import (
  "fmt"
  "net"
)

func lookup(host string) {
  fmt.Printf("%s:\n", host)
  addrs, _ := net.LookupHost(host)
  fmt.Printf("  %+v\n", addrs)
}

func main() {
  lookup("baseline-app.add-steps.svc.cluster.local")
  lookup("baseline-app")
}
```

Observe that the IPs from `baseline-app` are randomized, but `baseline-app.add-steps.svc.cluster.local` always begins with `10.48.4.193`:

```bash
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.0.199 10.48.3.163 10.48.3.162 10.48.2.13 10.48.0.198 10.48.2.14 10.48.2.12 10.48.2.11 10.48.2.10]
baseline-app:
  [10.48.2.10 10.48.0.199 10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.13 10.48.0.198 10.48.2.14 10.48.2.12 10.48.2.11]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11 10.48.0.199 10.48.0.198]
baseline-app:
  [10.48.0.198 10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11 10.48.0.199]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.2.12 10.48.2.11 10.48.2.10 10.48.0.199 10.48.3.163 10.48.3.162 10.48.2.13 10.48.0.198 10.48.2.14]
baseline-app:
  [10.48.2.14 10.48.2.12 10.48.2.11 10.48.2.10 10.48.0.199 10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.13 10.48.0.198]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.0.198 10.48.2.14 10.48.2.12 10.48.2.11 10.48.2.10 10.48.0.199 10.48.3.163 10.48.3.162 10.48.2.13]
baseline-app:
  [10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11 10.48.0.199 10.48.0.198]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.0.198 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11 10.48.0.199]
baseline-app:
  [10.48.0.199 10.48.0.198 10.48.4.193 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.2.11 10.48.0.199 10.48.0.198 10.48.3.163 10.48.3.162 10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13]
baseline-app:
  [10.48.3.162 10.48.2.13 10.48.0.198 10.48.2.14 10.48.2.12 10.48.2.11 10.48.2.10 10.48.0.199 10.48.4.193 10.48.3.163]
$ go run dnslookup.go
baseline-app.add-steps.svc.cluster.local:
  [10.48.4.193 10.48.2.11 10.48.2.10 10.48.0.199 10.48.3.163 10.48.3.162 10.48.2.13 10.48.0.198 10.48.2.14 10.48.2.12]
baseline-app:
  [10.48.2.12 10.48.2.10 10.48.2.14 10.48.2.13 10.48.2.11 10.48.0.199 10.48.0.198 10.48.4.193 10.48.3.163 10.48.3.162]
```